Dockerfile-go-deps

```diff
@@ -7,26 +7,31 @@
 
 FROM golang:1.9.4
 ENV TEMP_GOPATH=/temp-gopath
-ENV GOPATH=${TEMP_GOPATH}
-WORKDIR $GOPATH/src/github.com/runconduit/conduit
+WORKDIR ${TEMP_GOPATH}/src/github.com/runconduit/conduit
 
 # Download `dep` without being sensitive to changes to Gopkg.{toml,lock}.
 COPY bin/dep bin/dep
-RUN bin/dep version
+RUN GOPATH=${TEMP_GOPATH} bin/dep version
 
-# Vendor the Go dependencies.
+# Vendor the Go dependencies. `dep ensure` cacheds the entire Git repo for
```

Comment:
Typo here `cacheds` => `caches`

Reply:
> Typo here cacheds => caches

Thanks. I fixed it and re-ran bin/update-go-deps-shas.

Reply:
would you mind running `bin/dep ensure` and then `bin/update-go-deps-shas`? looks like i set an invalid `inputs-digest` in `Gopkg.lock` via https://github.com/runconduit/conduit/commit/33e3c3ace9c7e3d417707e97d2492547276177e2#diff-bd247e83efc3c45ae9e8c47233249f18R630